aten/src/ATen/native/native_functions.yaml

```diff
@@ -2842,6 +2842,29 @@
 - func: triplet_margin_loss(Tensor anchor, Tensor positive, Tensor negative, float margin=1.0, float p=2, float eps=1e-06, bool swap=False, int reduction=Mean) -> Tensor
   use_c10_dispatcher: full
 
+- func: true_divide.Tensor(Tensor self, Tensor other) -> Tensor
```

Comment:
I'm guessing you don't have an inplace function because the most common usage would be an error?

Reply:
We could definitely add an inplace (and method) variant of the function, but yes it seemed mostly a theoretical exercise. Especially since we'll eventually switch div itself to be true division, and div has all the variants we'd like.

Reply:
sounds like a good plan to me.